### DISCUSSION TITLE
How to deploy a finished project?

### DISCUSSION THREAD

**Discussion by ArchontBelketh** [unoplatform/uno#11332] [Q&A] [answered]
Hello.
I am a beginner in C# and UNO.
I have a finished project which runs well in Visual Studio 2022.
Is there a detailed step-by-step guide to help me deploy the application on my "server"(just computer with Apache 2.4). So that you can follow the url link and the application starts in the browser.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! We have [step by step deployment for Azure Static apps](https://platform.uno/docs/articles/guides/azure-static-webapps.html?q=static) that you can use as a based to deploy on your own environment.

You may also follow [parts of this documentation](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html) to help you with it.

**Reply by ArchontBelketh** [reply]
Thank you for answer!
